hack/local-monitor-testing/deploy_MDM_VM.sh

```diff
@@ -10,11 +10,13 @@ MDMFRONTENDURL=https://global.ppe.microsoftmetrics.com/
 MDMSOURCEENVIRONMENT=$LOCATION
 MDMSOURCEROLE=rp
 MDMSOURCEROLEINSTANCE=$HOSTNAME
+# use unique prefix for Azure resources when it is set, otherwise use your user's name
+export AZURE_PREFIX="${AZURE_PREFIX:-$USER}"
 
 echo "Using:"
 
 echo "Resourcegroup = $RESOURCEGROUP"
-echo "User          = $USER"
+echo "AzureUniquePrefix = $AZURE_PREFIX"
```

Comment:
nit: indentation for the other echo commands were using the same left padding.

Reply:
I do not fully understand what you mean by left padding. Let me know if the new commit helped resolve that.
(Here and other places)

Reply:
Just meant the spacing that kept the output aligned (all the values printed in the same column), but just a nitpick.